tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -11,19 +11,33 @@ namespace Datadog.Trace
     public static class Tags
     {
         /// <summary>
-        /// The environment of the instrumented service. It's value is usually constant for the lifetime of a process,
+        /// The environment of the instrumented service. Its value is usually constant for the lifetime of a process,
         /// but can technically change for each trace if the user sets it manually.
         /// This tag is added during MessagePack serialization using the value from <see cref="TraceContext.Environment"/>.
         /// </summary>
         public const string Env = "env";
 
         /// <summary>
-        /// The version of the instrumented service. It's value is usually constant for the lifetime of a process,
+        /// The version of the instrumented service. Its value is usually constant for the lifetime of a process,
         /// but can technically change for each trace if the user sets it manually.
         /// This tag is added during MessagePack serialization using the value from <see cref="TraceContext.ServiceVersion"/>.
         /// </summary>
         public const string Version = "version";
 
+        /// <summary>
+        /// The git commit hash of the instrumented service. Its value is usually constant for the lifetime of a process,
+        /// but can technically change for each trace if the user sets it manually.
+        /// This tag is added during MessagePack serialization using the value from <see cref="TraceContext.GitCommitSha"/>.
+        /// </summary>
+        public const string GitCommitSha = "_dd.git.commit.sha";
```

Comment:
Probably shouldn't be public (seems very unlikely customers would want to set it manually for each trace). I'd argue having it public makes it more likely they _will_ just because they don't realise they don't have to

Reply:
Fixed in 989c9c823ebb86da47f43eeaf48a2466a04e38af